README.md

```diff
@@ -4,13 +4,29 @@
 
```

Comment:
I think it's crucial for you to re-visit strings lesson. As one of primitive JS type, knowing how to work with strings effectively is very important. Please read feedbacks below, try to solve those problems, and really try to understand what each line of code is doing. Also try to avoid hard coding issue as I see many of them throughout different assignments. Again if you have any questions, let me or Alejandro know.